doc/release/upcoming_changes/22575.improvement.rst

```diff
@@ -0,0 +1,5 @@
+``np.pad`` with ``mode=wrap`` now returns expected results
+----------------------------------------------------------
+
+``np.pad`` with ``mode=wrap`` now always fills padding with 
+strict loops of original data.
```

Comment:
If we just fix the behavior, I think it is important to call this "compatibility" (or at least "change"), and also mention when it breaks:  We can only do this, if we expect that practically nobody uses this padding method when the pad size is larger than the initial array.

I haven't made up my mind yet.  If we introduce a `wrapmod`, we could also make an alias to wrap, but have `"wrap"` transition to an error when the wrap-width is too big?

Reply:
What specific action should I do to "make `modwap` an alias to `wrap`"?